### PR TITLE
Miscellaneous mappings, mostly world gen

### PR DESCRIPTION
**Pull request by IotaBread**
Maps most remaining unmapped classes and some unmapped methods, ports some Javadoc from Yarn

### CODE REVIEW

File: mappings/net/minecraft/world/gen/Density.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_ojoovffl net/minecraft/world/gen/Density
+	FIELD f_mpylalvy SURFACE D
+	FIELD f_oaliifne UNRECOVERABLY_DENSE D
```
Comment (NoComment1105) [inline mappings/net/minecraft/world/gen/Density.mapping]: Is `UNRECOVERABLY` a word? Surely something along the lines of `UNRECOVERABLE_DENSITY_LEVEL(S)` would be better in terms of coherent English. Not sure about the length though

**Review by NoComment1105 — CHANGES_REQUESTED** [review]
Looks good, just one comment

File: mappings/net/minecraft/world/gen/Density.mapping (hunk 1 above)
Comment (pluiedev) [reply mappings/net/minecraft/world/gen/Density.mapping]: language is constantly evolving ;)

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/block/EntityShapeContext.mapping [hunk 2]
```diff
@@ -8,5 +8,3 @@ CLASS net/minecraft/unmapped/C_qyrslpru net/minecraft/block/EntityShapeContext
 	METHOD <init> (Lnet/minecraft/unmapped/C_astfners;)V
 		ARG 1 entity
 	METHOD m_cgfxeuct getEntity ()Lnet/minecraft/unmapped/C_astfners;
-	METHOD m_dgjqzash (Lnet/minecraft/unmapped/C_xqketiuf;)Z
```
Comment (NoComment1105) [inline mappings/net/minecraft/block/EntityShapeContext.mapping]: This plus a few other files have the same changes in #81, not sure if this needs fixing here or if it is fine

**Review by NoComment1105 — APPROVED** [review]

File: mappings/net/minecraft/block/EntityShapeContext.mapping (hunk 2 above)
Comment (IotaBread) [reply mappings/net/minecraft/block/EntityShapeContext.mapping]: I don't think it causes merge conflicts if they are the same changes

Comment (NoComment1105) [reply mappings/net/minecraft/block/EntityShapeContext.mapping]: Alright 👍🏼

File: mappings/net/minecraft/world/level/ColorResolver.mapping [hunk 3]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_qodopjcw net/minecraft/world/level/ColorResolver
```
Comment (NoComment1105) [inline mappings/net/minecraft/world/level/ColorResolver.mapping]: This is mapped in #5, but I see comments about this on said PR, both have no response, so would indicate there is no plan to remove it from #5?

Comment (IotaBread) [reply mappings/net/minecraft/world/level/ColorResolver.mapping]: I was actually going to remove it here but I forgot

**Review by NoComment1105 — CHANGES_REQUESTED** [review]
`WorldGenRegistryDumpReport` is renamed already in #90, #93, #97

**Comment by IotaBread** [comment]
> WorldGenRegistryDumpReport is renamed already

Same thing as #96

**Review by NoComment1105 — APPROVED** [review]

**Review by triphora — APPROVED** [review]

**Pull request merged**